packages/azure-http-specs/specs/azure/resource-manager/large-header/mockapi.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  dyn,
+  dynItem,
+  json,
+  MockRequest,
+  passOnSuccess,
+  ScenarioMockApi,
+  ValidationError,
+} from "@typespec/spec-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+const SUBSCRIPTION_ID_EXPECTED = "00000000-0000-0000-0000-000000000000";
+const RESOURCE_GROUP_EXPECTED = "test-rg";
+const SIX_KB_STRING = randomString(1024 * 6);
```

Comment:
I am not sure whether randomness is required here. A char repeated 6k time should be good enough?

Reply:
Yeah, that should suffice. Let me update.